Libraries/UserInterface/src/server_main_window.cpp

```diff
@@ -293,15 +293,8 @@ void ServerMainWindow::OnDefaultAction()
 	}
 	else if (current_ui_ == LIST_PEERS)
 	{
-		LRESULT sel = ::SendMessage(listbox_, LB_GETCURSEL, 0, 0);
-		if (sel != LB_ERR)
-		{
-			LRESULT peer_id = ::SendMessage(listbox_, LB_GETITEMDATA, sel, 0);
-			if (peer_id != -1 && !callbacks_.empty())
-			{
-				std::for_each(callbacks_.begin(), callbacks_.end(), [&](MainWindowCallback* callback) { callback->ConnectToPeer(peer_id); });
-			}
-		}
+		// no work for us to do in this case, but do not fall through

```

Comment:
I just remember that this will break the HoloLens client since the only way to initialize the peer connection now is selecting the HoloLens client from the server main window. Can we keep it?

Reply:
oh. darn. let me see if we can solve the double selection case in a different way then, i will try

Reply:
please see https://github.com/CatalystCode/3DStreamingToolkit/pull/274/commits/a50701d03f81bec3d05df30cecb20342a9d20a5e for a better solution, i beleive